examples/kubernetes/snapshot/specs/snapshot-import/claim.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
Can you also update the `aws-ebs-csi-driver/examples/kubernetes/snapshot/README`?

Reply:
Sure, as the import snapshot is a different test case, should I put them in one README or separate it?

Reply:
I think we had different use cases on that README. Up to you.